### PR TITLE
[HCE]: Added release notes for HCE-0.4.2

### PR DESCRIPTION
**Pull request by Jonsy13**
Signed-off-by: Jonsy13 <vedant.shrotria@harness.io>

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/chaos-engineering/introduction/release-notes.md [hunk 1]
```diff
@@ -3,11 +3,43 @@ sidebar_position: 1
...
-# November 14, 2022
-
 Harness Chaos Engineering is updated regularly. Review the notes below for details about recent changes. 
 
-**Note:** Harness deploys updates progressively to different Harness cluster hosting accounts. The features and fixes in the release notes may not be available in your cluster immediately.
+:::note
+Harness deploys updates progressively to different Harness cluster hosting accounts. The features and fixes in the release notes may not be available in your cluster immediately.
+:::
+
+# December 2, 2022, 0.4.2
```
Comment (ajeshbaby) [inline docs/chaos-engineering/introduction/release-notes.md:12]: December 2, 2022, version 0.4.2

**Review by ajeshbaby — APPROVED** [review]

File: docs/chaos-engineering/introduction/release-notes.md [hunk 2]
```diff
@@ -3,11 +3,43 @@ sidebar_position: 1
...
+:::note
+Harness deploys updates progressively to different Harness cluster hosting accounts. The features and fixes in the release notes may not be available in your cluster immediately.
+:::
+
+# December 2, 2022, version 0.4.2
+
+## What’s New
+
+1. Update feature for ChaosHub enabling users to update details like Git Connector, Repository Name, Branch name & Name for an already connected ChaosHub.
+2. Adds CDN Support for Chaos Module Static Artifacts making UI load faster on client's devices.
```
Comment (neelanjan00) [inline docs/chaos-engineering/introduction/release-notes.md:17]:
```suggestion
2. Adds CDN Support for Chaos module static artifacts, making UI load faster on client's devices.
```

File: docs/chaos-engineering/introduction/release-notes.md [hunk 3]
```diff
@@ -3,11 +3,43 @@ sidebar_position: 1
...
+3. Added version Info in ChaosDriver & ChaosManager. Now, the versions will be available over provided endpoints `/chaos/driver/api/version` & `/chaos/manager/api/version` for ChaosDriver & ChaosManager respectively.
```
Comment (neelanjan00) [inline docs/chaos-engineering/introduction/release-notes.md:18]:
```suggestion
3. Added version info in ChaosDriver & ChaosManager. Now, the versions will be available over provided endpoints `/chaos/driver/api/version` & `/chaos/manager/api/version` for ChaosDriver & ChaosManager respectively.
```

File: docs/chaos-engineering/introduction/release-notes.md [hunk 4]
```diff
@@ -3,11 +3,43 @@ sidebar_position: 1
...
+4. Adds a range filter dropdown in the Experiment Runs Bar graph under Experiment overview allowing users to set the range on the last runs to be shown in the graph.
```
Comment (neelanjan00) [inline docs/chaos-engineering/introduction/release-notes.md:19]:
```suggestion
4. Adds a range filter dropdown in the **Experiment Runs** bar graph under Experiment overview allowing users to set the range on the last runs to be shown in the graph.
```

Comment (neelanjan00) [reply docs/chaos-engineering/introduction/release-notes.md:19]: Please embolden other names too

File: docs/chaos-engineering/introduction/release-notes.md [hunk 5]
```diff
@@ -3,11 +3,43 @@ sidebar_position: 1
...
+5. Adds support for all faults statuses in the Experiment Runs graph. Previously only `Failed` & `Passed` faults were getting shown, now faults in `Awaited`, `Stopped` & `N/A` states will also be available under the Experiment Runs Graph.
```
Comment (neelanjan00) [inline docs/chaos-engineering/introduction/release-notes.md:20]:
```suggestion
5. Adds support for all faults statuses in the Experiment Runs graph. Previously only `Failed` & `Passed` faults were getting shown, now faults in `Awaited`, `Stopped` & `N/A` states will also be available under the Experiment Runs graph.
```

File: docs/chaos-engineering/introduction/release-notes.md [hunk 6]
```diff
@@ -3,11 +3,43 @@ sidebar_position: 1
...
+1. Fixes Enterprise ChaosHub shown irrespective of searched terms by the users.
+2. Fixes httpProbe schema in UI to add support for new response timeout changes for HTTP probe. Now, probeTimeout for HTTP probes will be treated as response timeout & should be provided in seconds.
+3. Fixes the issue when the details of previously connected chaos infrastructure were getting pre-filled while connecting new chaos infrastructure.
+4. Fixes the Run button returning an error even when the Experiment run is already completed.
+5. Fixes calendar on the Experiments & Experiment Runs page having a default selection of 1 week. Now, all experiments & runs will be shown by default.
+6. Fixes panic error for k8sObjects and k8sLogs go-routines resulting in closed channel error.
+7. Fixes cancel(X) button & Back Button missing in Enable Chaos Infrastructure screen
+8. Fixes repeated error logs for ChaosHub in Chaos-Manager when it was not available to find some of the icons.
+9. Fixes the Expected Resilience Score changing to NaN when trying to override the same completely.
+10. Fixes resource-type not coming for aborting a Chaos Experiment in audit-trail
```
Comment (neelanjan00) [inline docs/chaos-engineering/introduction/release-notes.md:39]:
```suggestion
10. Fixes resource-type not coming for aborting a Chaos Experiment in audit-trail.
```

File: docs/chaos-engineering/introduction/release-notes.md [hunk 7]
```diff
@@ -3,11 +3,43 @@ sidebar_position: 1
...
+
+No early access features are available in this release.
+
+## Fixed Issues
+
+1. Fixes Enterprise ChaosHub shown irrespective of searched terms by the users.
+2. Fixes httpProbe schema in UI to add support for new response timeout changes for HTTP probe. Now, probeTimeout for HTTP probes will be treated as response timeout & should be provided in seconds.
+3. Fixes the issue when the details of previously connected chaos infrastructure were getting pre-filled while connecting new chaos infrastructure.
+4. Fixes the Run button returning an error even when the Experiment run is already completed.
+5. Fixes calendar on the Experiments & Experiment Runs page having a default selection of 1 week. Now, all experiments & runs will be shown by default.
```
Comment (neelanjan00) [inline docs/chaos-engineering/introduction/release-notes.md:34]:
```suggestion
5. Fixes calendar on the Experiments & Experiment Runs page having a default selection of one week. Now, all experiments & runs will be shown by default.
```

File: docs/chaos-engineering/introduction/release-notes.md [hunk 8]
```diff
@@ -3,11 +3,43 @@ sidebar_position: 1
...
+6. Fixes panic error for k8sObjects and k8sLogs go-routines resulting in closed channel error.
+7. Fixes cancel(X) button & Back Button missing in Enable Chaos Infrastructure screen
```
Comment (neelanjan00) [inline docs/chaos-engineering/introduction/release-notes.md:36]:
```suggestion
7. Fixes cancel(X) button & back button missing in Enable Chaos Infrastructure screen
```

**Review by neelanjan00 — APPROVED** [review]

**Review by ajeshbaby — APPROVED** [review]
LGTM

**Pull request merged**